airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/manifest.yaml

```diff
@@ -98,7 +98,7 @@ definitions:
     datetime_format: "%s"
     cursor_field: "{{ parameters.get('cursor_field',  'updated_at') }}"
     start_datetime:
-      datetime: "{{ timestamp(config.get('start_date')) | int or day_delta(-730, '%Y-%m-%dT%H:%M:%SZ') }}"
+      datetime: "{{ timestamp(config.get('start_date')) | int if config.get('start_date') else day_delta(-730, '%s') }}"
```

Comment:
Weird, did this bug out previously? Seems unrelated to concurrency work.

Reply:
we need to do this because i got rid of the extra python code `convert_config_to_declarative_stream_args()` so we're more declcarative and have less custom code.

And because we no longer automatically inject `start_date`, we can't just use `or` because the `None | int` would fail types or something along those lines.